erpnext/accounts/doctype/journal_entry/journal_entry.py

```
@@ -1681,15 +1704,15 @@ def get_account_details_and_party_type(account, date, company, debit=None, credi
-	reference_type=None,
-	reference_name=None,
-	debit=None,
-	credit=None,
-	exchange_rate=None,
+	posting_date: str,
+	account: str | None = None,
+	account_currency: str | None = None,
+	company: str | None = None,
+	reference_type: str | None = None,
+	reference_name: str | None = None,
+	debit: str | None = None,
```

Ruff (0.15.2)

[warning] Unused function argument: `debit`

(ARG001)